### PR TITLE
chore: support fs and s3 file provider

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from src.enums.providers import Providers
+from src.providers.s3Provider import S3Provider
+from src.providers.fSProvider import FSProvider
+from src.logger.jsonLogger import Logger
+import os
+import boto3
+    
+file_provider = os.environ.get('FILE_PROVIDER', 's3')
+source = os.environ.get('SOURCE_FILE', '/source/file/path')
+destination = os.environ.get('OUTPUT_DESTINATION', 'downloads')
```
Comment (asafMasa) [inline main.py:10]: make sure you have permissions to create this folder in open shift

File: src/providers/s3Provider.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import sys
+import os
+import boto3
+from src.logger.jsonLogger import Logger
+
+class S3Provider:
+
+    def __init__(self, destination):
+        self.endpoint_url = os.environ.get('S3_ENDPOINT_URL', 'http://localhost:9000')
```
Comment (asafMasa) [inline src/providers/s3Provider.py:9]: update readme to how to use each config

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good job

File: main.py (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply main.py:10]: fixed in 43b9c23

File: src/providers/s3Provider.py (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/providers/s3Provider.py:9]: fixed in 43b9c23

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,16 @@
 FROM python:3.6-alpine3.12
 RUN mkdir /app
 WORKDIR /app
-RUN mkdir downloads
+RUN mkdir /app/logs
```
Comment (asafMasa) [inline Dockerfile]: remove

**Review by asafMasa — APPROVED** [review]

File: Dockerfile (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply Dockerfile]: fixed in 1d9c3b7

**Pull request merged**